frontend/www/js/omegaup/typeahead.ts

```diff
@@ -251,8 +251,11 @@ export function groupTypeahead(
   cb?: CallbackType<{ label: string; value: string }>,
 ): void {
   if (!cb) {
-    cb = (event: Event, val: { label: string; value: string }) =>
-      $(<EventTarget>event.target).val(val.label);
+    cb = (event: Event, val: { label: string; value: string }) => {
+      $(<EventTarget>event.target).val(val.value);
+      $(<EventTarget>event.target).attr('data-alias', val.value);
+      $(<EventTarget>event.target).attr('data-name', val.label);
```

Comment:
```suggestion
      $(<EventTarget>event.target)
          .attr('data-value', val.value)
          .attr('data-label', val.label)
          .val(val.value);
```

también, por qué no es suficiente con cambiar esto a

```suggestion
      $(<EventTarget>event.target).val(val.value);
```

?

Reply:
Por algún motivo que no logro entender del todo, esto no le asigna el alias al input, de tal modo  que data-alias es de la forma que pude obtenerlo, los otros dos ya los quité porque no son necesarios

Reply:
oh ya veo, gracias por la explicación!

la razón es porque el jquery typeahead sólo puede exponer una cosa en el input. y no podemos ponerle el label como valor porque eso le mostraría el alias (y no el nombre) al usuario. entonces el uso de `data-value` es correcto.